mne/channels/layout.py

```diff
@@ -763,7 +766,7 @@ def plot(self, scale_factor=1.5, show_names=False):
                             show_names=show_names)
 
 
-def read_montage(kind, ch_names=None, path=None, scale=True):
+def read_montage(kind, ch_names=None, path=None, transform=False):
```

Comment:
should I leave the argument name the same?  scale isn't being used or
called internally by read_montage.

-teon

On Thu, Mar 12, 2015 at 2:10 PM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/channels/layout.py
> <https://github.com/mne-tools/mne-python/pull/1846#discussion_r26299289>:
>
> > @@ -763,7 +766,7 @@ def plot(self, scale_factor=1.5, show_names=False):
> >                              show_names=show_names)
> >
> >
> > -def read_montage(kind, ch_names=None, path=None, scale=True):
> > +def read_montage(kind, ch_names=None, path=None, transform=False):
>
> that's an API change
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/1846/files#r26299289>.
>